unstructured/partition/text.py

```diff
@@ -247,7 +247,7 @@ def partition_text(
     for ctext in file_content:
         ctext = ctext.strip()
 
-        if ctext:
+        if ctext and clean_bullets(ctext):
```

Comment:
can we then just pass in `clean_bullets(ctext)` to `element_from_text` and skip clean_bullets in the next function? Here we are calling the same regex pattern multiple times.

Reply:
So, I've moved the logic to `elements_from_text` but we can't remove the `clean_bullets` logic or maybe I'm misunderstood your suggestion.